package.json

```
@@ -6,7 +6,8 @@
     "amp:validate": "wait-on http://localhost:7080/news/articles/c9rpqy7pmypo.amp && amphtml-validator http://localhost:7080/news/articles/c9rpqy7pmypo.amp",
     "audit:ci": "audit-ci -low",
     "bbcA11y": "bbc-a11y",
-    "build": "rm -rf build && NODE_ENV=production webpack",
+    "build": "rm -rf build && NODE_ENV=production APP_ENV=test webpack && NODE_ENV=production APP_ENV=live webpack",
```

```suggestion
    "build": "export  NODE_ENV=production && rm -rf build && APP_ENV=test webpack && APP_ENV=live webpack",
```

Can't decide if this is more explicit or more confusing, but it's a suggestion either way...